Source/Components/FilePreviewPanel.cpp

```
@@ -44,6 +44,24 @@ std::vector<float> generateWaveformFromAudio(const std::vector<float>& samples,
     return waveform;
 }
 
+std::string truncateToWidth(NUIRenderer& renderer, const std::string& text, float fontSize, float maxWidth) {
+    if (maxWidth <= 0.0f) return "";
```

Cppcheck (2.20.0)

[style] The function 'isValid' is never used.

(unusedFunction)